components/breadcrumb/stories/breadcrumb.stories.js

```diff
@@ -48,13 +61,252 @@ Default.args = {
 		},
 		{
 			label: "Trend",
+		},
+		{
+			label: "January 2019 Assets",
+		},
+	],
+};
+
+export const MultilineNested = Template.bind({});
+MultilineNested.args = {
+	items: [
+		{
+			iconName: "FolderOpen",
+		},
+		{
+			label: "Sub Item",
+		},
+		{
+			label: "Trend",
+			isDragged: true,
+		},
+		{
+			label: "January 2019 Assets",
+		},
+	],
+	variant: "multiline",
+};
+MultilineNested.storyName = "Multiline, nested";
+MultilineNested.tags= ["!dev"];
+MultilineNested.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * Breadcrumbs can have optional behavior to allow for drag and drop functionality.
+ */
+export const Dragged = Template.bind({});
+Dragged.args = {
+	items: [
+		{
+			label: "Nav root",
+		},
+		{
+			label: "Trend",
+			isDragged: true,
+		},
+		{
+			label: "January 2019 Assets",
+		},
+	],
```

Comment:
This same setting seems to be repeated a lot below - do we want to either deconstruct the `Default` args:

```suggestion
    ...Default.args,
```

Reply:
Good call! I cleaned up that stories file and fixed all of the sentence case issues (I was following the docs site too closely!): https://github.com/adobe/spectrum-css/pull/2793/commits/26a6353de48dbfe7fd8ec5f99705609761e613a4